### PR TITLE
Add basic HTTP collector

### PR DESCRIPTION
**Pull request by arjunrn**
Refactored the PodJSON collector so that it can also be used into other collectors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -585,3 +585,60 @@ you need to define a `key` or other `tag` with a "star" query syntax like
...
+        name: http
+        selector:
+          matchLabels:
+            identifier: unique-metric-name
+      target:
+        averageValue: 1
+        type: AverageValue
+```
+
+The HTTP collector similar to the Pod Metrics collector. The metric name should always be `http-json`.
```
Comment (vetinari) [inline README.md:635]: this does not match the yaml example above, it it intended?

Comment (arjunrn) [reply README.md]: Fixed

**Comment by mikkeloscar** [comment]
:+1:

**Comment by arjunrn** [comment]
:+1:

**Pull request merged**